ui/pages/snaps/snap-view/snap-view.test.js

```diff
@@ -40,7 +40,7 @@ describe('SnapView', () => {
       renderWithProvider(<SnapView />, mockStore);
 
     // Snap name & Snap authorship component
-    expect(getAllByText('BIP-44 Test Snap')).toHaveLength(2);
+    expect(getAllByText('BIP-44 Test Snap')).toHaveLength(4);
```

Comment:
?

Reply:
This update was required because I updated the mocked state here: https://github.com/MetaMask/metamask-extension/pull/25175/files#diff-56c8fa46f8b1b6aadfa7b9ce3652c38a6770efc37593832725bcc6615b3be9faR1944

And then because of how permission names (descriptions) are constructed, more of these elements are found:
![Screenshot 2024-06-27 at 14 12 48](https://github.com/MetaMask/metamask-extension/assets/13301024/38e9064e-64e4-4097-967a-52dc4a2aeae9)

_Permissions I added to the mocked state are adding now an extra two `BIP-44 Test Snap`._

Reply:
Ah, makes sense!